client/favorites.jsx

```diff
@@ -43,7 +35,10 @@ export default class Favorites extends React.Component {
     })
       .then(response => response.json())
       .then(data => {
-        this.setState({ isLoading: false });
+        this.setState({ isLoading: false, removalId: removalId, favoritesData : this.state.favoritesData});
+        setTimeout(function() {
+          location.reload()
+        } ,2000)
```

Comment:
Nooo, it's impolite to just refresh the page if you can avoid it. What problem are you trying to solve? Do you need to filter your favorites array?